core/src/main/scala/org/apache/spark/api/python/PythonRDD.scala

```diff
@@ -159,7 +159,12 @@ private[spark] object PythonRDD extends Logging {
    * @return 2-tuple (as a Java array) with the port number of a local socket which serves the
    *         data collected from this job, and the secret for authentication.
    */
-  def collectAndServe[T](rdd: RDD[T]): Array[Any] = {
+  def collectAndServe[T](
```

Comment:
I'll let @ueshin judge, but my guess is that, even though this is a "private" API, we'll want to add a new collectAndServe with the 2 arguments, leaving the old one in case 3rd-party libraries use the private API.